inductiva/__init__.py

```diff
@@ -26,6 +26,10 @@
 logs.setup(getattr(logging, os.environ.get("INDUCTIVA_LOG_LEVEL", "INFO")))
 
 api_url = os.environ.get("INDUCTIVA_API_URL", "https://api.inductiva.ai")
+_input_dir = contextvars.ContextVar("INDUCTIVA_INPUT_DIR",
+                                    default=os.environ.get(
+                                        "INDUCTIVA_INPUT_DIR",
+                                        "inductiva_input"))
```

Comment:
Has this been discussed with @ClaudiaRSilva?
I don't see why the inputs should be placed outside the `inductiva_output` folder. The purpose of this folder is to store everything that gets generated and downloaded through the inductiva package. It is not meant to store only the outputs of simulations. With the introduction of the  `inductiva_input` folder, you're separating the inputs from the outputs. Perhaps you could have the inputs/outputs downloaded to `inductiva_output/<task_id>/[inputs|outputs/...]`

Please clarify this design decision before moving forward.

cc @hpenedones, @ClaudiaRSilva